frontend/src/metabase/query_builder/components/expressions/ExpressionEditorTextfield/helper-text-strings.ts

```diff
@@ -1,5 +1,11 @@
 import { t } from "ttag";
-import { HelpText } from "./types";
+import moment from "moment-timezone";
+import MetabaseSettings from "metabase/lib/settings";
+import { formatTime } from "metabase/lib/formatting";
+import type { Database } from "metabase-types/types/Database";
+import { HelpText } from "metabase-lib/expressions/types";
+
+const reportTimezone = MetabaseSettings.get("report-timezone-long");
```

Comment:
This won't work if you go to Admin, change the timezone and come back. It should be read within a function.